tools/nni_cmd/tensorboard_utils.py

```diff
@@ -42,7 +42,8 @@ def copy_data_from_remote(args, nni_config, trial_content, path_list, host_list,
         local_path = os.path.join(temp_nni_path, trial_content[index].get('id'))
         local_path_list.append(local_path)
         print_normal('Copying log data from %s to %s' % (host + ':' + path_list[index], local_path))
-        sftp = create_ssh_sftp_client(host, machine_dict[host]['port'], machine_dict[host]['username'], machine_dict[host]['passwd'])
+        sftp = create_ssh_sftp_client(host, machine_dict[host]['port'], machine_dict[host]['username'], machine_dict[host]['passwd'],
+                                      machine_dict[host]['sshKeyPath'], machine_dict[host]['passphrase'])
```

Comment:
what if `machine_dict[host]` do not have `sshKeyPath` field?

Reply:
Oh sorry...

How about modify line 40
https://github.com/microsoft/nni/blob/6b375d6d1e402bad020ac213edf39aec6ad7b11d/tools/nni_cmd/tensorboard_utils.py#L39-L46

to 
```python
machine_dict[machine['ip']] = {'port': machine['port'], 'passwd': machine['passwd'], 'username': machine['username'], 'sshKeyPath': machine.get('sshKeyPath'), 'passphrase': machine.get('sshKeyPath')}
```

Reply:
sure

Reply:
> sure

done.